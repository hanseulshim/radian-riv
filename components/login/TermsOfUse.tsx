import React from 'react'

export default function TermsOfUse() {
  return (
    <div className="terms-of-use">
      <h4>Terms of Use:</h4>
      <object
        data={`${process.env.baseUrl}/TermsOfUse.pdf`}
        type="application/pdf"
        width="100%"
        height="300px"
      />
    </div>
  )
}
