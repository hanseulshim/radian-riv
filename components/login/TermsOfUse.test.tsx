import * as React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import TermsOfUse from './TermsOfUse'

describe('TermsOfUse', () => {
  let component
  beforeAll(() => {
    component = shallow(<TermsOfUse />)
  })
  it('has terms-of-use container', () => {
    expect(component.find('.terms-of-use')).toHaveLength(1)
  })
  it('has terms of use title', () => {
    expect(component.find('h4').text()).toEqual('Terms of Use:')
  })
  it('has object to display pdf', () => {
    expect(component.find('object').prop('data')).toContain('TermsOfUse.pdf')
  })
  it('should match snapshot', () => {
    expect(toJson(component)).toMatchSnapshot()
  })
})
