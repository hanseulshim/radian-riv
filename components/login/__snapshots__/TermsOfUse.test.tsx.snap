// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TermsOfUse should match snapshot 1`] = `
<div
  className="terms-of-use"
>
  <h4>
    Terms of Use:
  </h4>
  <object
    data="undefined/TermsOfUse.pdf"
    height="300px"
    type="application/pdf"
    width="100%"
  />
</div>
`;
